README.md

```diff
@@ -1,3 +1,108 @@
+Well, I did and I could not find a good tutorial on the full process to get the spark job running, so here is mine.
+Included in this article and accompanying repository should be everything you need to take your Scala Spark job and get it running on AWS [Data Pipeline](https://aws.amazon.com/datapipeline/) and [EMR](https://aws.amazon.com/emr/).
+
+## Code Repo
+
+This tutorial is not going to walk you through the process of actually writing your specific Scala Spark job to do whatever number crunching you need. 
+There are [plenty of](https://www.analyticsvidhya.com/blog/2017/01/scala/) [resources](https://spark.apache.org/docs/latest/quick-start.html) [available](https://www.coursera.org/learn/scala-spark-big-data) already to get you started on that. 
+The code template for setting up a Spark Scala job is available in [this GitHub repo](https://github.com/jessebrizzi/spark-fat-jar-template).
+
+Assuming that you have already written your Spark Job locally in something like [IntelliJ](https://www.jetbrains.com/idea/) or in the cloud with something like [Zeppelin](https://zeppelin.apache.org/) and [AWS EMR](https://aws.amazon.com/emr/) and you are only using the [AWS Java SDK](https://aws.amazon.com/sdk-for-java/) to connect to your AWS data stores, you should be able to drop your code in the [Main](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala#L4) function of [SparkJob.scala](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala) and and run the [deploy.sh](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh) script to upload the fat jar to your S3 bucket. 
+
+If not, then it may take some extra work on your part. 
+To run a Scala Spark job on AWS you need to compile a [fat jar](https://stackoverflow.com/questions/19150811/what-is-a-fat-jar) that contains your source code for your job and all of the libraries it needs to run.
+This project already has the [sbt-assembly](https://github.com/sbt/sbt-assembly) plugin setup and a [`assemblyMergeStrategy`](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/build.sbt#L13) set up to package the Spark, Hadoop, and AWS SDK together in the fat jar.
+If you need to add in other libraries that do not play well with each other, or you are using a noncompatible version of Spark for this current repo, there are a few good [resources](http://queirozf.com/entries/creating-scala-fat-jars-for-spark-on-sbt-with-sbt-assembly-plugin) available to help you through the needed `build.sbt` changes.
+
+Outside of the needed changes to [SparkJob.scala](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/src/main/scala/SparkJob.scala) and maybe [build.sbt](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/build.sbt), you need to set a few parameters in the [deploy.sh](https://github.com/jessebrizzi/spark-fat-jar-template/blob/master/deploy.sh) script. 
```

Comment:
Can you provide any hint on the possible needed changes to build.sbt that you foresee?